package.xml

```diff
@@ -18,9 +18,9 @@
   <buildtool_depend>ament_cmake_python</buildtool_depend>
   <buildtool_depend>python_cmake_module</buildtool_depend>
 
-  <build_depend>builtin_interfaces</build_depend>
-  <build_depend>rclcpp</build_depend>
-  <build_depend>rclpy</build_depend>
+  <depend>builtin_interfaces</depend>
+  <depend>rclcpp</depend>
```

Comment:
`rclcpp` isn't actually a `build_depend`, though it is required in the headers and in the tests.  So I think this should probably be:

```
<build_export_depend>rclcpp</build_export_depend>
<test_depend>rclcpp</test_depend>
```

Reply:
Agreed.